build.ps1

```diff
@@ -597,22 +597,23 @@ function Build-WiXProject() {
 
   $Properties = $Properties.Clone()
   TryAdd-KeyValue $Properties Configuration Release
-  TryAdd-KeyValue $Properties BaseOutputPath "$($Arch.BinaryCache)\msi\"
+  TryAdd-KeyValue $Properties BaseOutputPath "$($Arch.BinaryCache)\installer-scripts\"
```

Comment:
I really like this!  Or even `installer` perhaps?

Reply:
Sure!

Reply:
Updated in latest